kyo-core/shared/src/test/scala/kyoTest/varsTest.scala

```diff
@@ -45,7 +45,7 @@ class varsTest extends KyoTest:
                         assert(outerValue == 1)
                     }
             }
-        }
+        }.pure
```

Comment:
Interesting that this was compiling before, it shouldn't.

Reply:
Did this only happen after adding the `Int` type?

Reply:
Actually I could have removed the `Int` annotations. It was my initial attempt but then I saw `pure` was missing.